tests/misc/tests/misc/misc.ts

```diff
@@ -22,6 +22,10 @@ const { assert, expect } = require("chai");
 const nativeAssert = require("assert");
 const miscIdl = require("../../target/idl/misc.json");
 
+const TOKEN_2022_PROGRAM_ID = new anchor.web3.PublicKey(
+  "TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA"
```

Comment:
This is not the Token 2022 program id, it's the standard SPL Token's [program id](https://github.com/solana-labs/solana-program-library/blob/ec671eb7f3a98fbe12c27fec89093f66ce3f7e65/token/program/src/lib.rs#L80). Token 2022's program id is [TokenzQdBNbLqP5VEhdkAS6EPFLC1PHnBqCXEpPxuEb](https://github.com/solana-labs/solana-program-library/blob/ec671eb7f3a98fbe12c27fec89093f66ce3f7e65/token/program-2022/src/lib.rs#L94) which means tests didn't change functionally with these changes.

Reply:
🤦🏼‍♂️